src/js/containers/awardV2/AwardV2Container.jsx

```diff
@@ -144,6 +188,12 @@ export class AwardContainer extends React.Component {
 AwardContainer.propTypes = propTypes;
 
 export default connect(
-    (state) => ({ award: state.awardV2 }),
-    (dispatch) => bindActionCreators(awardActions, dispatch)
+    (state) => ({ award: state.awardV2, isDownloadPending: state.bulkDownload.download.pendingDownload }),
+    (dispatch) => ({
+        setDownloadExpectedUrl: (url) => dispatch(setDownloadExpectedUrl(url)),
```

Comment:
Any rationale for removing `bindActionCreators`? We've also use a pattern of creating a `combinedActions` object that gets passed to `bindActionCreators` in other containers (e.g. [here](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/containers/search/SearchSidebarSubmitContainer.jsx#L123))

Reply:
Mainly out of convenience. Was having some trouble cause I'm not using an alias when importing the AC's and Im not sure how elegant I could use `Object.assign` to combine all of them?

Reply:
Also was getting this error when trying to use it in the same way as the docs [here](https://react-redux.js.org/using-react-redux/connect-mapdispatch).

![image](https://user-images.githubusercontent.com/12897813/56443644-90dc1d00-62c3-11e9-9cb3-b21efba095a6.png)

I think we should try and squeeze in some babel config so we can use the spread operator. The benefit of this would be instead of doing `Object.assign()` which requires the parameters to be objects (?), we can use the spread operator which is a bit easier IMO. Really don't understand why it's not already in there based on the current config... 

The error above is about using `redux-thunk` which would be a super small config change and a new dependency, but would allow for some more flexibility around implementing different patterns etc...